Src/MoneyFox.Ui/Views/Settings/SettingsViewModel.cs

```diff
@@ -31,4 +41,33 @@ public CurrencyViewModel SelectedCurrency
     }
 
     public IReadOnlyList<CurrencyViewModel> AvailableCurrencies { get; }
+
+    public string SelectedAccount
+    {
+        get => selectedAccount;
+
+        set
+        {
+            SetProperty(field: ref selectedAccount, newValue: value);
+            settingsFacade.DefaultAccount = selectedAccount;
+            OnPropertyChanged();
+        }
+    }
+
+    public void LoadAccounts()
+    {
+        availableAccounts = mediator.Send(new GetAccountsQuery()).Result.Select(a => a.Name).ToList();
```

Comment:
@NPadrutt I see using VM is overkill and using name alone serves our purpose. Let me know if I am missing anything here. When should we need a VM?

Reply:
@balaji119  if we just want to display some information I would tend to agree. In this case I think it would make sense to use one together with the Account ID and then use the Account Id in the settings to select the account. This would have the advantage that the id is fix and can't be changed. So even when an account is renamed the default account still can be selected.

Reply:
Added view model with ID and Name.